_episodes/26.md

```diff
@@ -0,0 +1,272 @@
+---
+layout: episode
+title:  "26: Trino discovers data catalogs with Amundsen"
+date: 2021-09-16
+tags: data governance catalog discovery metadata 
+youtube_id: "rNI5fCU74M0"
+wistia_id: "mnr5t7vx0g"
+sections: 
+   - title: "Concept of the week"
+     desc: "Data discovery and Amundsen"
+     time: 724
+   - title: "Concept of the week"
+     desc: "Amundsen Architecture"
+     time: 1450
+   - title: "Concept of the week"
+     desc: "Amundsen as a subcomponent to data mesh"
+     time: 2720
+   - title: "PR of the week"
+     desc: "Index Trino Views"
+     time: 3606
+   - title: "Question of the week"
+     desc: "Can I add a UDF without restarting Trino?"
+     time: 4374
+---
+
+## Guests
+ * Mark Grover, Co-creator of Amundsen and Founder at [Stemma](https://www.stemma.ai/)
+ ([@mark_grover](https://twitter.com/mark_grover)).
+ 
+## Release 362
+
+Official announcement items from Martin:
+Not yet available since release it not yet out… but soon.
+
+Manfreds notes:
+* Add new listagg function contributed by Marius
+* Join performance and DISTINCT performance improvements
+* SQL security related changes in ALTER SCHEMA
+* Add “IN table” for CREATE/DROP/… ROLE
+* Whole bunch of improvements in the BigQuery connector
+* Numerous improvements for Parquet file usage in Hive connector
+* All connector docs now have SQL support section
+
+## Concept of the week: Data discovery and Amundsen
+
+Data discovery is the process that aids in the analysis of data where siloed 
+data has been centralized, and it is difficult to find data or overlap between
+disparate data sets. Many teams have their own view of the world when it comes 
+to the data they need, but they commonly need to reason about how their data 
+relates to data outside of their domain. 
+
+Questions about who owns the data to help identify individuals responsible for 
+maintaining the standards, documentation, and cleanliness of the data is useful
+to identify who to call for help or collaboration. This ultimately allows 
+analysts to discover patterns in, audit and question existing policies, and 
+ultimately have a system of record and shared front end around their data 
+policies.
+
+### What is Amundsen?
+
+Amundsen provides data discovery by using ETL processes to scrape metadata from
+all of the data sources, and making a central location to search your data.
```

Comment:
sources. It creates a central location to collect all that metadata and enables search and other analytics of this metadata.